.changeset/tidy-boats-rescue.md

```diff
@@ -0,0 +1,6 @@
+---
+"@navikt/ds-react": minor
+"@navikt/ds-css": minor
+---
+
+Modal: New prop for aligning modal to top
```

Comment:
```suggestion
Modal: New prop `placement` for anchoring modal to the top or center of viewport. 
```

Reply:
I think it sounds like anchoring to the center is a new feature (and is it anchoring when centered?)
Counter-suggestion:
```suggestion
Modal: New prop `placement` for anchoring the modal to the top of the viewport.
```

Reply:
Or do you think we should mention `center` somehow? 🤔

Reply:
Can omit "center" 👍 Anchoring might be to technical, so "aligning" or "placing" works 2.